torchinductor/codegen/wrapper.py

```diff
@@ -296,10 +296,24 @@ def generate(self):
         result = IndentedBuffer()
         result.splice(self.header)
         result.splice(self.prefix)
+
+        def extract_name(out):
+            o = out
+            while hasattr(o, "data"):
+                if hasattr(o, "name"):
+                    return o.name
+                else:
+                    o = o.data
+            return ""
+
+        out_names = [extract_name(out) for out in V.graph.graph_outputs]
```

Comment:
We collect names of all outputs here:
https://github.com/pytorch/torchdynamo/blob/da8bb9a875beede7b721425087d06ebae7bc76f8/torchinductor/scheduler.py#L849
and, here:
https://github.com/pytorch/torchdynamo/blob/da8bb9a875beede7b721425087d06ebae7bc76f8/torchinductor/scheduler.py#L600

I'd suggest adding a function like:
```
V.graph.get_output_names()
````

That combines the 3 places